2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_setup.py:_flush():76] Loading settings from C:\Users\jaydenpark\.config\wandb\settings
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_setup.py:_flush():76] Loading settings from C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\settings
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_init.py:_log_setup():428] Logging user logs to C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\run-20230315_093909-3ibk3exb\logs\debug.log
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_init.py:_log_setup():429] Logging internal logs to C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\run-20230315_093909-3ibk3exb\logs\debug-internal.log
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_init.py:_jupyter_setup():378] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000265FD3C0A00>
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_init.py:init():462] calling init triggers
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0003, 'data_size': 1, 'batch_size': 128, 'model_name': 'cardiffnlp/twitter-xlm-roberta-base-sentiment'}
2023-03-15 09:39:09,226 INFO    MainThread:27020 [wandb_init.py:init():515] starting backend
2023-03-15 09:39:09,226 INFO    MainThread:27020 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2023-03-15 09:39:09,226 INFO    MainThread:27020 [backend.py:ensure_launched():217] starting backend process...
2023-03-15 09:39:09,274 INFO    MainThread:27020 [backend.py:ensure_launched():222] started backend process with pid: 26188
2023-03-15 09:39:09,274 INFO    MainThread:27020 [wandb_init.py:init():525] backend started and connected
2023-03-15 09:39:09,290 INFO    MainThread:27020 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2023-03-15 09:39:09,290 INFO    MainThread:27020 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-15 09:39:09,290 INFO    MainThread:27020 [wandb_init.py:init():596] updated telemetry
2023-03-15 09:39:09,290 INFO    MainThread:27020 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2023-03-15 09:39:10,703 INFO    MainThread:27020 [wandb_run.py:_on_init():1924] communicating current version
2023-03-15 09:39:10,737 INFO    MainThread:27020 [wandb_run.py:_on_init():1928] got version response upgrade_message: "wandb version 0.14.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-15 09:39:10,737 INFO    MainThread:27020 [wandb_init.py:init():662] starting run threads in backend
2023-03-15 09:39:10,992 INFO    MainThread:27020 [wandb_run.py:_console_start():1898] atexit reg
2023-03-15 09:39:10,992 INFO    MainThread:27020 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2023-03-15 09:39:10,992 INFO    MainThread:27020 [wandb_run.py:_redirect():1808] Wrapping output streams.
2023-03-15 09:39:10,999 INFO    MainThread:27020 [wandb_run.py:_redirect():1832] Redirects installed.
2023-03-15 09:39:10,999 INFO    MainThread:27020 [wandb_init.py:init():688] run started, returning control to user process
2023-03-15 09:39:11,000 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:11,000 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:11,010 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 09:39:11,012 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:11,012 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:11,027 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 09:39:11,400 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:11,400 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:11,416 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 09:39:11,416 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:11,416 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:11,431 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 09:39:11,431 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:11,431 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:11,447 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 09:39:11,447 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:11,447 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:11,462 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 09:39:11,462 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:11,462 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:11,478 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 09:39:11,655 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:11,655 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:11,671 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 09:39:26,098 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 09:39:26,098 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 09:39:26,115 INFO    MainThread:27020 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-15 10:09:19,402 INFO    MainThread:27020 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-15 10:09:19,402 INFO    MainThread:27020 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-15 10:19:26,766 INFO    MainThread:27020 [wandb_run.py:_atexit_cleanup():1867] got exitcode: 0
2023-03-15 10:19:26,768 INFO    MainThread:27020 [wandb_run.py:_restore():1839] restore
2023-03-15 10:19:27,350 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 592
}

2023-03-15 10:19:27,631 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 592
}

2023-03-15 10:19:28,502 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 8662
}

2023-03-15 10:19:28,611 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 8662
}

2023-03-15 10:19:28,720 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 8662
}

2023-03-15 10:19:28,836 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 8662
}

2023-03-15 10:19:28,937 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 592
  total_bytes: 8662
}

2023-03-15 10:19:29,047 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8662
  total_bytes: 8662
}

2023-03-15 10:19:29,156 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8662
  total_bytes: 8662
}

2023-03-15 10:19:29,266 INFO    MainThread:27020 [wandb_run.py:_on_finish():1996] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8662
  total_bytes: 8662
}

